src/ONYXKEYS.js

```diff
@@ -42,6 +42,12 @@ export default {
     // Has information about the network status (offline/online)
     NETWORK: 'network',
 
+    // Contains all the personalDetails of the user
+    PERSONAL_DETAILS: 'personalDetails',
```

Comment:
Why do we need to add this? Can't we just use `withCurrentUserPersonalDetails`?

Reply:
Should be good now. We actually use only `withCurrentUserPersonalDetails`. I removed this key